tensorflow/contrib/framework/python/ops/script_ops.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Script Language Operators. See the @{$python/script_ops} guide.
+
+@@py_func
+"""
+
+# pylint: disable=g-bad-name
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.python.framework import tensor_shape
+from tensorflow.python.util import nest
+
+from tensorflow.python.ops.script_ops import py_func as _py_func
```

Comment:
Please use the hidden ops for that.

Reply:
What do you mean with `hidden ops`? I used a renaming because of the name collision.

Reply:
@drpngx this is fine -- it's a contrib-only extension of py_func, py_func still exists as a public symbol, so we don't need hidden_ops.